docs/product/issues/issue-priority/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Issue Priority
+sidebar_order: 31
+description: "Learn about issue priority."
+---
+
+<Note>
+
+This feature is only available to [Early Adopters](/product/accounts/early-adopter-features/) at this time.
+
+</Note>
+
+Issue priority helps you identify and address critical errors impacting your application's functionality and user experience. By assigning a priority level to each issue, Sentry can surface the most pressing matters first, allowing your development team to focus on resolving impactful problems efficiently.
+
+## How Priority Works
+
+There are three discrete priority levels for issues in Sentry:
+
+- **High**: Issues that are likely to be actionable and require immediate attention.
+- **Medium**: Issues that are likely to be actionable and require attention in the near future.
+- **Low**: Issues that do not require immediate attention.
+
+### Priority Assignment on Issue Creation
+
+When Sentry receives an event, it assigns a priority to the issue based on the event's log level (for error issues) or severity (non-error issues like performance). If the event's severity is high or the log level is `error` or `fatal`, the issue is assigned a high priority. If the severity is medium or the log level is `warning`, the issue is assigned a medium priority. If the severity is low or the log level is `info` or `debug`, the issue is assigned a low priority.
+
+### Automatic Priority Updates
+
+Sentry continuously monitors the volume of events for each issue. If an issue experiences a surge in occurrences (escalates), its priority will be automatically bumped to reflect its growing significance. See the [escalating issues algorithm](/product/issues/states-triage/escalating-issues/) for more details on how escalations are identified.
+
+When an issue is no longer escalating, its priority will revert to the previous level.
+
+### Manual Priority Updates
+
+You can manually update the priority of an issue at any time, either from the issue details page or from the issue stream. Once an issue's priority has been manually updated, it will no longer be automatically adjusted based on event volume.
```

Comment:
```suggestion

You can update issue priority manually at any time, either from the issue details page or from the issue stream. But once an issue's priority has been changed manually, it will no longer be automatically adjusted if the event volume escalates.
```